sdk/core/azure-core/tests/test_tracing_implementations.py

```diff
@@ -162,3 +163,26 @@ def test_add_attribute(self):
             wrapped_class.add_attribute("test", "test2")
             assert wrapped_class.span_instance.attributes["test"] == "test2"
             assert parent.attributes["test"] == "test2"
+
+    def set_http_attributes(self):
```

Comment:
is this test getting picked up? I think by default `pytest` looks for a `test` prefix (would be conventional to have in any case)

Reply:
you are right. Thats my bad. Fixed it